prometheus/operator/base/cluster-metrics-servicemonitor.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  name: cluster-monitor-federation
+  labels:
+    monitor-component: cluster-resources
+    team: opendatahub
+spec:
+  endpoints:
+    - honorLabels: true
+      params:
+        'match[]':
+          - '{__name__=~".+"}'
```

Comment:
I'm sorry I keep nitting on things :sweat_smile: . Do you know specifically what metrics we are using for the dashboards?
Ideally we'd only federate those over from cluster monitoring. This way we could also reduce the scrape intervals

Reply:
@anishasthana added the specific metrics that need to be federated :+1:

Reply:
does 30s seem reasonable?